src/components/tips/tip-template.tsx

```diff
@@ -22,14 +22,13 @@ import analytics from '@/utils/analytics'
 
 const TipTemplate = ({
   tip,
-  tips,
   coursesFromTag,
 }: {
   tip: Tip
-  tips: Tip[]
   coursesFromTag: any
 }) => {
   const markComplete = trpc.tips.markTipComplete.useMutation()
+  const {data: tips = []} = trpc.tips.published.useQuery()
```

Comment:
in this case we don't need to load that list of tips server side, might as well load it in on the client and let them see the page

I think you can do this with Suspense to and get the "streaming" response, but I'm not sure how to accomplish that yet but it would involve dropping `use client` and thinking about it differently.